cmd/exporters/influxdb/influxdb.go

```diff
@@ -112,7 +113,8 @@ func (e *InfluxDB) Init() error {
 
 		urlToUSe := "http://" + *addr + ":" + strconv.Itoa(*port)
 		url = &urlToUSe
-		e.url = fmt.Sprintf("%s/api/v%s/write?org=%s&bucket=%s&precision=%s", *url, *version, *org, *bucket, *precision)
+		e.url = fmt.Sprintf("%s/api/v%s/write?org=%s&bucket=%s&precision=%s", *url, *version, util.EscapeUrl(*org),
```

Comment:
does the EscapeUrl function carry enough weight to add or can we call url.PathEscape directly?

Reply:
i cant directly use url.PathEscape since url is already consumed by  influxdb.go as a variable. hence i coudlnt directly use it . otherwise i have to rename e.url everywhere seems like

Reply:
sure. will add unit test for this.